docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -40,15 +40,15 @@ Currently, Codacy sends coverage data to your Git provider from both the old and
       <td>February 14th, 2024</td>
       <td>Deprecation of the old Coverage status check</td>
       <td>
-        <p>From this day on, the new checks will no longer be marked <strong>beta</strong> and will be the official status checks for Codacy Coverage. The old check will be marked <strong>deprecated</strong> but will keep working.</p>
+        <p>From this day on, the new checks will no longer be marked <strong>beta</strong> and will be the official status checks for Codacy Coverage. The old check will be marked <strong>deprecated</strong> but will keep working.</p><!--NOTE Temporary comment for review purpose, to remove-->
         <p>If on this date you are still using the old check to block merging pull requests on GitHub, please <a href="#if-you-are-using-the-old-coverage-status-check-to-block-merging-pull-requests-on-github">update your Git provider settings</a>.</p>
       </td>
     </tr>
     <tr>
       <td>April 18th, 2024</td>
       <td>Brownout of the old Coverage status check</td>
       <td>
-        <p>Codacy will not send the old status check on this day.</p>
+        <p>Codacy will not send the old status check on this day.</p><!--NOTE Temporary comment for review purpose, to remove-->
         <p>This will help you confirm that your setup has been updated correctly and lower the risk of disruptions when the old status check is removed.</p>
```

Comment:
How? What are the steps to confirm if the setup has been updated correctly or not?

Reply:
How about this? Unfortunately, I don't have more details about "how" it would break. It's just meant to break for one day if the new setup is not correct.

```suggestion
        <p>If, on this day, you can merge your pull requests using the new checks and don't encounter other issues, it means that your setup has been updated correctly.</p>
```

Reply:
I would keep the sentence describing the goal and would add how can they confirm.
WDYT?

```suggestion
        <p>This will help you confirm that your setup has been updated correctly and lower the risk of disruptions when the old status check is removed.<br/>If on this day you can merge your pull requests using the new checks and don't encounter other issues, it means that your setup has been updated correctly. Otherwise, if on this date you are still using the old check to block merging pull requests on GitHub, please <a href="#if-you-are-using-the-old-coverage-status-check-to-block-merging-pull-requests-on-github">update your Git provider settings</a>.</p>
```

Reply:
LGTM 👍